spec/function.dd

```diff
@@ -56,7 +56,7 @@ $(GNAME InOut):
     $(D lazy)
     $(D out)
     $(D ref)
-    $(RELATIVE_LINK2 return-ref-parameters, $(D return ref))
+    $(RELATIVE_LINK2 return-ref-parameters, $(D return))
```

Comment:
The link to return-ref-parameters is now confusing since it can equally well mean return-scope now, maybe remove it?